master/internal/rm/kubernetesrm/kubernetes_resource_manager_intg_test.go

```diff
@@ -851,6 +853,142 @@ func TestVerifyNamespaceExists(t *testing.T) {
 	require.Error(t, err)
 }
 
+func TestCreateNamespaceHelper(t *testing.T) {
```

Comment:
I think it's ok to call this `TestCreateNamespace` even though it only implicitly calls the helper!